Course_id still in resource table after being dropped by conversion

## Describe the bug (Tell us what happens instead of the expected behavior) :

@chrisrrowland noticed this error when running on an empty database

`student_dashboard  | [2020-08-13 08:22:17 -0400] [ERROR] [models.py:268] Couldn't fetch Resource list in Course 17700000000335188 due to: Cannot resolve keyword 'course_id' into field. Choices are: id, name, resource_id, resource_type, resourceaccess`

This wasn't an issue in my database but resource for some reason had the course_id column still. It looks like pandas `to_sql` was adding this back in. We'll need to fix this so it's not created anymore and the query works.

--- a/dashboard/cron.py
+++ b/dashboard/cron.py
@@ -363,13 +363,8 @@ class DashboardCronJob(CronJobBase):
                          f'{resource_access_df.dtypes}')
 
             # Make resource data from resource_access data
-            resource_df = resource_access_df.copy(deep=True)
-            resource_df = resource_df.drop(
-                columns=['user_id', 'access_time', 'course_id'])
-            resource_df = resource_df.drop_duplicates('resource_id')
-
-            # set resource_id as index for pangres.upsert
-            resource_df: pd.DataFrame = resource_df.set_index('resource_id')
+            resource_df = resource_access_df.filter(["resource_id", "resource_type", "name"])
+            resource_df = resource_df.drop_duplicates(["resource_id"])
 
             logger.debug(f'resource_df:\n'
                          f'{resource_df}\n'

--- a/dashboard/models.py
+++ b/dashboard/models.py
@@ -263,7 +263,7 @@ class ResourceQuerySet(models.QuerySet):
         :return:
         """
         try:
-            return list(self.values_list('resource_type', flat=True).distinct().filter(course_id=course_id))
+            return list(self.values_list('resource_type', flat=True).distinct().filter(resourceaccess__course_id=course_id))
         except(self.model.DoesNotExist, Exception) as e:
             logger.error(f"Couldn't fetch Resource list in Course {course_id} due to: {e}")
             return None

